### ISSUE TITLE
Vaccum clean room no more work HA 2023.6.1

### ISSUE THREAD

**Issue by skycordellaa** [al-one/hass-xiaomi-miot#1150]
launching a script like this
```yaml
aspira_cucina:
  alias: Aspira cucina
  sequence:
  - service: xiaomi_miot.set_property
    data:
      entity_id: vacuum.ijai_v3_b0ec_robot_cleaner
      field: vacuum.mode
      value: 0
    enabled: false
  - service: xiaomi_miot.call_action
    data:
      entity_id: vacuum.ijai_v3_b0ec_robot_cleaner
      siid: 7
      aiid: 3
      params:
      - '14'
      - 0
      - 1
  mode: single
```

result an error in log

2023-06-08 13:45:45.821 ERROR (MainThread) [homeassistant.components.script.aspira_cucina] Aspira cucina: Error executing script. Unexpected error for call_service at pos 2: Passing coroutines is forbidden, use tasks explicitly.
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 452, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 685, in _async_call_service_step
    await service_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1910, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1950, in _execute_service
    await cast(Callable[[ServiceCall], Awaitable[None]], handler.job.target)(
  File "/config/custom_components/xiaomi_miot/__init__.py", line 422, in async_service_handler
    await asyncio.wait(update_tasks)
  File "/usr/local/lib/python3.11/asyncio/tasks.py", line 415, in wait
    raise TypeError("Passing coroutines is forbidden, use tasks explicitly.")
TypeError: Passing coroutines is forbidden, use tasks explicitly.
2023-06-08 13:45:45.836 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/script/__init__.py", line 477, in _async_run
    return await self.script.async_run(script_vars, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 1533, in async_run
    await asyncio.shield(run.async_run())
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 410, in async_run
    await self._async_step(log_exceptions=False)
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 454, in _async_step
    self._handle_exception(
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 477, in _handle_exception
    raise exception
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 452, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 685, in _async_call_service_step
    await service_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1910, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1950, in _execute_service
    await cast(Callable[[ServiceCall], Awaitable[None]], handler.job.target)(
  File "/config/custom_components/xiaomi_miot/__init__.py", line 422, in async_service_handler
    await asyncio.wait(update_tasks)
  File "/usr/local/lib/python3.11/asyncio/tasks.py", line 415, in wait
    raise TypeError("Passing coroutines is forbidden, use tasks explicitly.")
TypeError: Passing coroutines is forbidden, use tasks explicitly.

**Comment by skycordellaa** [comment]
plese delete this issues is opened with wrong account

**Comment by lirtm** [comment]
y delete? I have same error

**Comment by antoweb** [comment]
> y delete? I have same error

Because inhave used a wrong github account. I want reioennwith another account

**Comment by al-one** [comment]
Fixed in `v0.7.10`, please upgrade.

**Comment by lirtm** [comment]
> Fixed in `v0.7.10`, please upgrade.

Yes, All good, all work. Merci!
